knimin/handlers/barcode_util.py

```diff
@@ -172,8 +173,35 @@ def _build_email(self, login_user, barcode, email_type,
         return subject, body_message
 
 
+@set_access(['Scan Barcodes'])
+class PushQiitaHandler(BaseHandler):
+    @authenticated
+    def get(self):
+        barcodes = db.get_unsent_barcodes_from_qiita_buffer()
+        status = db.get_send_qiita_buffer_status()
+        dat = {'status': status, "barcodes": barcodes}
+        self.write(json_encode(dat))
+        self.finish()
+
+    @authenticated
+    def post(self):
```

Comment:
Do we need some kind of check to know that there is no other browser sending already?

Reply:
Unclear. Will make a note, but doesn't prohibit testing the larger interaction so would prefer to not resolve this question within this pull request

Reply:
Thanks, add issue so we can have resolution?

Reply:
See the mutex TODO above